app/helpers/users_helper.rb

```diff
@@ -1,2 +1,9 @@
 module UsersHelper
+    # Returns the Gravatar for the given user.
+  def gravatar_for(user, options = { size: 80 })
```

Comment:
This method does more than one task, make it adhere to Single Responsibility Principle